pytorch_lightning/trainer/trainer.py

```diff
@@ -826,6 +835,12 @@ def _validate_impl(
         # --------------------
         Trainer._log_api_event("validate")
         self.verbose_evaluate = verbose
+        # log hyper-parameters
+        if self.logger is not None:
+            # save exp to get started (this is where the first experiment logs are written)
+            # self.logger.log_hyperparams(self.lightning_module.hparams_initial)
+            self.logger.log_graph(self.lightning_module)
+            self.logger.save()
```

Comment:
Is this a hard requirement for you? Because this also changes behaviour for all other backends, which we usually want to avoid.